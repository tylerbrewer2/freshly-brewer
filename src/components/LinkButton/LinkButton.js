import React from 'react';

import Button from '../Button';

const LinkButton = Button.withComponent('a');

export default LinkButton;