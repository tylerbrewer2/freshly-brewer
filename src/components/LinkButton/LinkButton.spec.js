import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import LinkButton from './LinkButton';

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <LinkButton to="/">Test</LinkButton>
    </MemoryRouter>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
