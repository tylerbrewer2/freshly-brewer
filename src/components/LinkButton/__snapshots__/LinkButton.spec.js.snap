exports[`test renders correctly 1`] = `
<a
  className="sc-bwzfXH cRSDhw"
  href="/"
  onClick={[Function]}>
  Test
</a>
`;
